### DISCUSSION TITLE
Move 'Standard Notes Backups' folder from home directory into .config folder in Linux build.

### DISCUSSION THREAD

**Discussion by thewh1teagle** [standardnotes/forum#2710] [Suggestions] [answered]

**Comment by arielsvg** [comment, marked as answer]
Hey @thewh1teagle, the backups folder is meant to be visible and separate from app configuration, because we expect users to rely on it if something goes wrong.

**Comment by arielsvg** [comment]
I'll move your issue to the forum since this is the intended behavior :)
